client/src/store/modules/profile/actions.js

```diff
@@ -0,0 +1,52 @@
+import uploadFileService from '@/services/upload/fileService';
+import requestService from '@/services/requestService';
+import { UPDATE_BRANDING_LOGO } from './types/mutations';
+
+export default {
+    async updatePassword(context, password, oldPassword) {
+        try {
+            const response = await requestService.patch('/profile/', {
+                password: password,
+                oldPassword: oldPassword
+            });
+
+            return Promise.resolve(response?.data?.data);
+        } catch (error) {
+            return Promise.reject(error);
+        }
+    },
+
+    async saveBranding({ commit }, logo) {
+        try {
+            const response = await uploadFileService.upload(logo);
+
+            const url = response?.data?.data?.logo?.url;
+
+            await requestService.patch('/profile/', { logo: url });
+
+            commit(UPDATE_BRANDING_LOGO, url);
+
+            return Promise.resolve(response?.data?.data);
+        } catch (error) {
+            return Promise.reject(error);
```

Comment:
this Promise.reject is useless, because without this try catch it will work the same. I mean this:

```
async f() {
    try {
        const response = await uploadFileService.upload(logo);

        return Promise.resolve(response?.data?.data);
    } catch (error) {
         return Promise.reject(error);
    }
}
```
the same as this:

```
async f() {
        const response = await uploadFileService.upload(logo);

        return response?.data?.data;
}
```